industrial_taxonomy/pipeline/glass_embed/flow.py

```diff
@@ -48,19 +61,14 @@ def start(self):
             list(glass_companies_house_lookup().keys())
         ).to_list()
 
-        org_descriptions = org_descriptions.loc[self.org_ids]
-
-        if self.test_mode and not current.is_production:
-            test_size = 200_000
-            org_descriptions = org_descriptions.head(test_size)
-            self.org_ids = self.org_ids[:test_size]
+        nrows = 50_000 if self.test_mode and not current.is_production else None
+        self.org_ids = self.org_ids[:nrows]
 
-        org_descriptions = org_descriptions["description"].to_list()
+        self.org_descriptions = org_descriptions.loc[self.org_ids][
```

Comment:
:lady_beetle:  This isn't restricted to `nrows` !

One of the benefits of the below is that we take `nrows` descriptions once on the dataframe and then decompose into the keys and values which means we always end up with them being the same length.
https://github.com/nestauk/industrial_taxonomy/blob/a9e9db6a0f2e4a5f50f8ec4c897ed2dc9b971b69/industrial_taxonomy/pipeline/glass_description_ngrams/nlp_flow.py#L88-L97